src/pages/hyeriShin/Main/Feeds.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+
+class Feeds extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      reply: '',
+      comment: [],
+    };
+  }
+
+  uesrsCommentState = e => {
+    this.setState({
+      reply: e.target.value,
+    });
+  };
+
+  uesrsCommentAdd = e => {
+    e.preventDefault();
+    this.setState({
+      comment: this.state.comment.concat(this.state.reply),
```

Comment:
```suggestion
      comment: [...comment, reply],
```
- 구조분해 할당 해주시고
- 스프레드 연산자 사용할 수도 있겠네요